helpers/queries/jupiter-lend-ethena.sql

```
@@ -0,0 +1,295 @@
+    left join jupiter_solana.vaults_call_init_vault_config as v
+        on p.protocol = v.account_vault_config
+    left join jupiter_solana.flashloan_call_init_flashloan_admin as f
+        on p.protocol = f.account_flashloan_admin
+)
+
+-- Get unique tokens per protocol type with their start date
+, juplend_protocol_tokens as (
+    select protocol_type, mint, min(block_date) as block_date
+    from (
+        select protocol_type, supply_mint as mint, block_date from juplend_protocols
+        union
```

SQLFluff (4.1.0)

[error] 'UNION [DISTINCT|ALL]' is preferred over just 'UNION'.

(AM02)